src/platform-apps/channels/channel-list.test.tsx

```diff
@@ -16,10 +17,10 @@ describe('ChannelList', () => {
 
   it('renders each channel name', () => {
     const channels = [
-      { id: 'one', name: 'first channel' },
-      { id: 'two', name: 'second channel' },
-      { id: 'three', name: 'third channel' },
-      { id: 'four', name: 'fourth channel' },
+      { id: 'one', name: 'first channel', category: 'catg1' },
+      { id: 'two', name: 'second channel', category: 'catg1' },
+      { id: 'three', name: 'third channel', category: 'catg2' },
+      { id: 'four', name: 'fourth channel', category: 'catg2' },
```

Comment:
i don't think we need to add categories to tests that aren't using them. let's remove the category from these, since this test doesn't have anything to do with categories.